README.md

```diff
@@ -47,7 +47,7 @@ In order to ensure that your end users always have a functioning version of your
 - Android (4.1+)
 - Windows (UWP)
 
-We try our best to maintain backwards compatability of our plugin with previous versions of React Native, but due to the nature of the platform, and the existence of breaking changes between releases, it is possible that you need to use a specific version of the CodePush plugin in order to support the exact version of React Native you are using. The following table outlines which CodePush plugin versions officially support the respective React Native versions:
+We maintain backwards compatability of our plugin with previous versions of React Native. Due to the nature of the platform, and the existence of breaking changes between releases, it is possible that you need to use a specific version of the CodePush plugin in order to support the exact version of React Native you are using. The following table outlines which CodePush plugin versions officially support the respective React Native versions:
```

Comment:
I personally think the original flows a little better. Feels like more of a cohesive thought than with the two difference sentences.